EdgeMultiplay/Scripts/DataContracts.cs

```diff
@@ -679,6 +679,7 @@ public class Player
         /// helper instance variable for serializing and deserializing playerTags
         /// </summary>
         [DataMember (EmitDefaultValue = false)]
+#pragma warning disable 0649
```

Comment:
What does this do?

Reply:
Just to disable the Warning message "Player Tags is never used" in Unity Console Logs